src/main/kotlin/jambda/camp/clean/domain/Order.kt

```diff
@@ -0,0 +1,11 @@
+package jambda.camp.clean.domain
+
+import java.util.*
+
+data class Order(
+    val deliveryDate: OrderDeliveryDate
```

Comment:
Muss ```Order``` nicht noch mit dem ```ShoppingCart``` verbunden werden? Sonst weiss man nicht welche Produkte in die Bestellung kommen.